### PR TITLE
Ports "Majorly buffs standard holoparas"

### PR DESCRIPTION
**Pull request by gumner**
dank memes

### CODE REVIEW

File: code/modules/mob/living/simple_animal/guardian/types/standard.dm [hunk 1]
```diff
@@ -16,15 +16,28 @@
...
-	if(isliving(target))
-		src.say("[src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry][src.battlecry]!!")
-		playsound(loc, src.attack_sound, 50, 1, 1)
-		playsound(loc, src.attack_sound, 50, 1, 1)
-		playsound(loc, src.attack_sound, 50, 1, 1)
-		playsound(loc, src.attack_sound, 50, 1, 1)
-	..()
+/mob/living/simple_animal/hostile/guardian/punch/AttackingTarget(mob/living/T, mob/living/carbon/human/H)
+	if(istype(T))
+		var/holopunchrng = null
```
Comment (Carbonhell) [inline code/modules/mob/living/simple_animal/guardian/types/standard.dm:26]: wtf are those 3 lines...?
just do switch(rand(1,100))... no?

File: code/modules/mob/living/simple_animal/simple_animal.dm [hunk 2]
```diff
@@ -277,13 +280,32 @@
...
+			var/new_y = P.starting.y + pick(0, 0, 0, 0, 0, -1, 1, -2, 2)
+			var/turf/curloc = get_turf(src)
+
+			P.original = locate(new_x, new_y, P.z)
+			P.starting = curloc
+			P.current = curloc
+			P.firer = src
+			P.yo = new_y - curloc.y
+			P.xo = new_x - curloc.x
+			P.Angle = ""
```
Comment (Carbonhell) [inline code/modules/mob/living/simple_animal/simple_animal.dm:299]: this line feels wrong, you sure angle is a string...?

Comment (gumner) [reply code/modules/mob/living/simple_animal/simple_animal.dm:299]: it is,or was before rebase uness they changed it,i tested it before rebase and worked *shrug

**Comment by gumner** [comment]
holy fuck i cant do a commit for some reason i will try to open a new pr

**Pull request closed without merging**